index.html

```diff
@@ -228,6 +234,38 @@ <h1>
       </ul>
     </div>
   </section>
+
+  <section class="contact">
+
+    <div>
+     <h2 class="contact-text">Contact me</h2>
+     <div class="par-container"><p class="par-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p></div>
+    </div> 
+    <div>
+     <form action="/my-handling-form-page" method="post">
```

Comment:
- [ ] Please according to the requirement, kindly sign up to [formspree service](https://formspree.io/html/), so that you can have a action value similar to `https://formspree.io/f/{form_id}`. The action value shouldn't be `/my-handling-form-page` but the `url` generated from formspree. ℹ️